### ISSUE TITLE
Could not start portable version anymore

### ISSUE THREAD

**Issue by virtualdreams** [RandomEngy/VidCoder#1003]
### Problem Description

Could not start `VidCoder-7.13-Portable.exe`.

### What version of VidCoder are you running?

7.13

### Encode Log

```text
VidCoder 7.12 SquirrelInstaller
System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\MyUsername\AppData\Local\Temp\packages\RELEASES'.
```

**Comment by RandomEngy** [comment]
You get that error after you launch the portable version? Is there anything else to the error? Usually it has some more information below it.

I'm wondering if it's incorrectly concluded that it's the installer version...

**Comment by virtualdreams** [comment]
Yes, I got the error message right after the start. After the error there was an exception, which was not copied. Unfortunately, the logs from yesterday are gone and I can no longer reproduce the error.
The only thing I did, in the folder "AppData/Local/Temp" tried to delete everything and the computer was then in sleep mode.

**Comment by RandomEngy** [comment]
Did it break after you deleted everything in the temp folder? Or was it fixed after you deleted everything in the temp folder?

The portable installer works by extracting all the files needed to run to that temp folder (under RarSFX0), then running the program from there. When the program exits it deletes those files.

**Comment by virtualdreams** [comment]
It only worked after I cleared the temp folder. The folder was not created at all.

In the meantime I tried to create "/packages/RELEASES" in the hope that it is a rights problem.

**Comment by RandomEngy** [comment]
Thanks, that's good to know. Reported to the creators of the installer library: https://github.com/clowd/Clowd.Squirrel/issues/63 . Will also work on a workaround to get the correct value even that `Update.exe` is there.

**Comment by RandomEngy** [comment]
Fixed in 7.14.
